src/WelcomeBody.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import "./WelcomeBody.css";
+
+export default function WelcomeBody(props) {
```

Comment:
I don't like this name when there is already a Body component. This makes it sound like a sibling to Body but this is actually a child of the Body. 

WelcomePage
LoginScreen
SplashScreen etc. 

Read online to see if there are other names to such a component.

Reply:
Changed to WelcomeScreen